configuration/cdap-site.xml

```diff
@@ -1047,6 +1082,139 @@
     </value-attributes>
   </property>
 
+  <!-- Messaging System Configuration -->
+
+  <property>
+    <name>messaging.container.instances</name>
+    <value>1</value>
+    <description>
+      Number of instances for the messaging service
+    </description>
+    <final>true</final>
+  </property>
+
+  <property>
+    <name>messaging.container.memory.mb</name>
```

Comment:
Add this to `themes.json` along with `num.cores`... Just copy another container config.